---
spatial_db_msgs/ObjectInstance[] objects
